games/telewave/telewave.py

```diff
@@ -0,0 +1,196 @@
+"""
+    Read each other's minds
+"""
+import asyncio
+import discord
+import json
+import random
+from collections import deque
+from chowder import chowder
+
+with open("games/telewave/telewave_config.json", "r") as read_file:
+    config = json.load(read_file)
+
+
+class TelewaveTeam:
+    def __init__(self, players, name, score):
+        self.guessers = deque(players)
+        self.psychic = self.guessers.popleft()
+        self.score = score
+        self.name = name
+
+    def to_string(self):
+        return f"{self.psychic.name}, {', '.join([g.name for g in self.guessers])}"
+
+    def mention_guessers(self):
+        return f"{', '.join([g.mention for g in self.guessers])}"
+
+    def mention(self):
+        return f"{self.psychic.mention}, {', '.join([g.mention for g in self.guessers])}"
+
+    def rotate_psychic(self):
+        self.guessers.append(self.psychic)
+        self.psychic = self.guessers.popleft()
+
+    def get_players(self):
+        return [self.psychic] + list(self.guessers)
+
+
+async def start(bot, ctx, players):
+    await ctx.send(f"Starting a game of **telewave** with {', '.join([p.mention for p in players])}")
+    random.shuffle(players)
+    team_names = get_team_names()
+
+    # Per official Wavelength rules - the team going second starts with a 1 point lead
+    team1 = TelewaveTeam(players[:len(players)//2], name=team_names[0], score=0)
+    team2 = TelewaveTeam(players[len(players)//2:], name=team_names[1], score=1)
+    winners = await play(bot, ctx, team1, team2)
+
+    await ctx.send(f"GG, **{winners.name}** wins")
+    return winners.get_players()
+
+
+async def play(bot, ctx, team1, team2):
+    max_score = config["max_score"]
+    going_again = False
+    while team1.score < max_score and team2.score < max_score:
+        await wait(ctx, going_again, team1)
+        prompt = get_prompt()
+        answer = random.randint(0, 100)
```

Comment:
decimals?

Reply:
Thought it'd be simpler for answers/submissions to only be integers. The average of the guesses is still calculated as a decimal